src/model/enhance-numbered-lists.ts

```diff
@@ -177,6 +177,25 @@ const makeThumbnailsRound = (elements: CAPIElement[]): CAPIElement[] => {
 	return inlined;
 };
 
+const isItemLink = (element: CAPIElement): boolean => {
+	if (!element) return false;
+	// Checks if this element is a 'item link' based on the convention: <ul> <li>...</li> </ul>
+	if (
+		element._type !== 'model.dotcomrendering.pageElements.TextBlockElement'
+	) {
+		return false;
+	}
+	const frag = JSDOM.fragment(element.html);
+	if (!frag || !frag.firstElementChild) return false;
+
+	const hasULWrapper = frag.firstElementChild.nodeName === 'UL';
+	const hasOnlyOneChild = frag.firstElementChild.childElementCount === 1;
+	const hasLINestedWrapper =
+		frag.firstElementChild?.firstElementChild?.nodeName === 'LI';
+
+	return hasULWrapper && hasOnlyOneChild && hasLINestedWrapper;
+};
+
```

Comment:
nothing changes here, `isItemLink` just needs to be defined before it is being used (it is used in `addH3s`)